### ISSUE TITLE
Bug: visualization API table package issue in IE7

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-issues#1878]
```
What steps will reproduce the problem? Please provide a link to a
demonstration page if at all possible, or attach code.
1.use any sample code example to draw a table



What component is this issue related to (DataTable, DataView, etc)?


Are you using the test environment (version 1.1)?
(If you are not sure, answer NO)
NO


What operating system and browser are you using?

Windows 7, IE7

*********************************************************
For developers viewing this issue: please click the 'star' icon to be
notified of future changes, and to let us know how many of you are
interested in seeing it resolved.
*********************************************************
The error we get on draw is
Object doesn.t support property or method querySelector

This was working fine for us till last night in IE7. WE have not made any changes to
our application. Kindly suggest a solution.


```

Original issue reported on code.google.com by `neeshmary` on 2015-02-19 18:57:23

**Comment by orwant** [comment]
```
This will be fixed in the next v41 candidate release.  We are rolling back the broken
version now.
```

Original issue reported on code.google.com by `dlaliberte@google.com` on 2015-02-19 19:10:55

**Comment by orwant** [comment]
```
Thank You for your quick response!

Would it be possible to let me know when the roll back will occur and also when v41
is planned?

```

Original issue reported on code.google.com by `neeshmary` on 2015-02-19 19:15:38

**Comment by orwant** [comment]
```
The broken v41 release is now being rolled back, and should finish in a few hours. 
The next candidate v41 will very likely go out early next week.
```

Original issue reported on code.google.com by `dlaliberte@google.com` on 2015-02-19 19:31:08

**Comment by orwant** [comment]
```
We have this problem  too, glad to hear there is a quick fix for this.
```

Original issue reported on code.google.com by `prabu.gopalakrishnan` on 2015-02-19 21:42:23

**Comment by orwant** [comment]
```
Can this be a duplicate of issue 1868 we opened this morning and the rollback will also
resolve our problem?
```

Original issue reported on code.google.com by `meob63` on 2015-02-19 22:07:24

**Comment by orwant** [comment]
```
No the table sizing issues are different.  The rollback would have restored things to
the previous state, but the rollback failed.  The fixes we have so far will not change
the table behavior.  You may get what you want by including the option { width: '100%'
}, but there may be other issues with the table chart.
```

Original issue reported on code.google.com by `dlaliberte@google.com` on 2015-02-19 22:16:06

**Comment by orwant** [comment]
```
We are gettting the same error message, "Object doesn.t support property or method querySelector"
 on WIN IE8 and WIN IE9

We tried to explicityly load the rolled back version like this:

google.load('visualization', '1.0', {'packages':['table']});

but it is still outputting the same error message.
```

Original issue reported on code.google.com by `prabu.gopalakrishnan` on 2015-02-19 22:17:13

**Comment by orwant** [comment]
```
We are not having issue with the table size, but actually loading the table.  We get
the java script error message, and no loaded table.  Is there any other workaround
you can suggest?
```

Original issue reported on code.google.com by `prabu.gopalakrishnan` on 2015-02-19 22:20:41

**Comment by orwant** [comment]
```
The querySelector problem will be fixed by the changes that hopefully will be deployed
this evening.

Loading '1.0' would load the last stable version, assuming it is actually stable. 
But at this time, loading 1.0 will load v41 with its bugs, and we can't rollback (this
time around).  So you just have to wait until the fixed versions shows up later this
evening.
```

Original issue reported on code.google.com by `dlaliberte@google.com` on 2015-02-19 22:25:34

**Comment by orwant** [comment]
```
We are using this functionality in a healthcare setting, and it effects patient care.
 We would appreciate if solution was found quickly.  Thank You.
```

Original issue reported on code.google.com by `prabu.gopalakrishnan` on 2015-02-19 22:33:18

**Comment by orwant** [comment]
```
Hi,

Could you please update us once the rollback is complete. A large set of users are
waiting on this to continue business.

We would like to verify the entire application to check nothing else is broken.

Thanks!
```

Original issue reported on code.google.com by `neeshmary` on 2015-02-20 02:36:11

**Comment by orwant** [comment]
```
Please update us on the status of rollback effort. Thank you.
```

Original issue reported on code.google.com by `gary.gumenyuk` on 2015-02-20 03:46:58

**Comment by orwant** [comment]
```
Was something rolled out at approximately 9:45 pm ET and then pulled back out maybe
around 10:30 pm ET? It looked like the issues were fixed, and now api appears broken
again. Were you doing a quick test of the rolled back package before officially releasing?

```

Original issue reported on code.google.com by `bcantana` on 2015-02-20 03:51:05

**Comment by orwant** [comment]
```
Well, the rollback actually failed, but undoing the rollback is succeeding.  After it
finishes, then a new set of fixes will be applied which should fix the querySelector
issue.  It is difficult to say more at this time.
```

Original issue reported on code.google.com by `dlaliberte@google.com` on 2015-02-20 04:46:47

**Comment by orwant** [comment]
```
Was the version rolled back yet?
```

Original issue reported on code.google.com by `erik.m.hoffman` on 2015-02-20 13:05:16

**Comment by orwant** [comment]
```
Was there any CSS Changes? Our table header is not displaying at all and tables are
misplaced? What changes we need to make in our end?
```

Original issue reported on code.google.com by `arunprasath.ramachandran` on 2015-02-20 13:45:41
